libfastpm/thermalvelocity.c

```diff
@@ -331,7 +331,7 @@ void
 fastpm_split_ncdm(FastPMncdmInitData * nid,
         FastPMStore * src,
         FastPMStore * dest,
-        int every,
+        int every,   ///////////////can removE!!!!!!!!!!!!!!!!!
```

Comment:
Remove?

Reply:
The 'every' arg is no longer needed here because now src is the unsplit ncdm, whose np has already accounted for 'every'.

Reply:
In that case, just remove it.

Reply:
Yeah, in future I'll make the code cleaner before I commit.

Reply:
It is usually more efficient catching these things from another pair of eyes rather than trying to grind the code yourself. That's the whole point of reviewing the pull request.